README.rst

```diff
@@ -105,47 +114,15 @@ As **MAPIE** is compatible with the standard scikit-learn API, you can see that
 - How easy it is **to wrap your favorite scikit-learn-compatible model** around your model.
 - How easy it is **to follow the standard sequential** ``fit`` and ``predict`` process like any scikit-learn estimator.
 
-.. code:: python
-
-    # Uncertainty quantification for regression problem
-    import numpy as np
-    from sklearn.linear_model import LinearRegression
-    from sklearn.datasets import make_regression
-    from sklearn.model_selection import train_test_split
-
-    from mapie.regression import MapieRegressor
-
-
-    X, y = make_regression(n_samples=500, n_features=1)
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5)
-
-    regressor = LinearRegression()
-
-    mapie_regressor = MapieRegressor(estimator=regressor, method='plus', cv=5)
-
-    mapie_regressor = mapie_regressor.fit(X_train, y_train)
-    y_pred, y_pis = mapie_regressor.predict(X_test, alpha=[0.05, 0.32])
-
-.. code:: python
-
-    # Uncertainty quantification for classification problem
-    import numpy as np
-    from sklearn.linear_model import LogisticRegression
-    from sklearn.datasets import make_blobs
-    from sklearn.model_selection import train_test_split
-
-    from mapie.classification import MapieClassifier
-
-
-    X, y = make_blobs(n_samples=500, n_features=2, centers=3)
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5)
+Uncertainty quantification for a regression problem
```

Comment:
The titles "Uncertainty quantification for a regression problem" and classification are maybe a bit too big for the actual content. Maybe introducing the notebook in 2 bullet points would be more readable?